content/docs/guides/contribute/code-review.en.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Review process"
```

Comment:
We should discuss about it, because it's different thing to do a review or to submit a PR. I prefer write documentation for specific point of view. Am I a reviewer or a code contributor ? Things are not exactly the same. Anyway, I propose to discuss about it later if you're ok to let this as this for now ?

Reply:
It makes sense as you state it, both are OK for me.